src/pages.cpp

```diff
@@ -1993,13 +2103,7 @@ static void setup_command_list(GtkWidget *widget, Option *o)
                 opt_value     = executable;
             }
 
-            if (std::string(o->default_value.s) == "string")
-            {
-                dyn_opt->type = OPTION_TYPE_STRING;
-            } else
-            {
-                continue;
-            }
+            dyn_opt->type = OPTION_TYPE_STRING;
```

Comment:
What is the reason behind this change?

Reply:
There are no default values for dynamic lists, so this check is meaningless.